### ISSUE TITLE
Add ZFS support in installer

### ISSUE THREAD

**Issue by UWorkbench** [manjaro/release-plan#205]
Hey people, may we have ZFS support like Antergos?

**Comment by Chrysostomus** [comment]
Manjaro-architect has it. You wish this to be implemented in calamares as well?

**Comment by UWorkbench** [comment]
Yes, if possible. In architect can I choose the size of ZFS pool? I'm not seeing this option

**Comment by Chrysostomus** [comment]
Ooops, I read zsh instead of zfs. Zfs support is not yet implemented
